src/proc/mod.rs

```diff
@@ -212,7 +213,131 @@ impl crate::Binding {
     pub fn to_built_in(&self) -> Option<crate::BuiltIn> {
         match *self {
             Self::BuiltIn(bi) => Some(bi),
-            Self::Location(..) => None,
+            Self::Location { .. } => None,
+        }
+    }
+}
+
+impl super::Module {
```

Comment:
Let's move this to `src/proc/interpolator.rs` (or another name, but having a separate module).